src/app/interactions/ripple/ripple-stying/ripple-styling-sample.component.html

```diff
@@ -1,92 +1,27 @@
-                        <button igxButton="fab" igxButtonColor="yellow" igxButtonBackground="black" igxRipple>
-                            <igx-icon fontSet="material">add</igx-icon>
-                        </button>
-                    </div>
-                </div>
-            </article>
-        </section>
-        <section class="section">
-            <article class="sample-column">
-                <h5 class="sample-title">Icon Buttons (with overridden ripple colors)</h5>
-                <div class="buttons-sample">
-                    <div class="button-sample">
-                        <button igxButton="icon" igxButtonColor="white" igxButtonBackground="#E41D78" igxRipple="white"
-                            igxRippleCentered="true">
-                            <igx-icon fontSet="material">search</igx-icon>
-                        </button>
-                    </div>
-                    <div class="button-sample">
-                        <button igxButton="icon" igxButtonColor="#FBB13C" igxButtonBackground="#340068" igxRipple="#FBB13C"
-                            igxRippleCentered="true">
-                            <igx-icon fontSet="material">favorite</igx-icon>
-                        </button>
-                    </div>
-                    <div class="button-sample">
-                        <button igxButton="icon" igxButtonColor="yellow" igxButtonBackground="black" igxRipple="yellow"
-                            igxRippleCentered="true">
-                            <igx-icon fontSet="material">more_vert</igx-icon>
-                        </button>
-                    </div>
-                </div>
-            </article>
-    
-            <article class="sample-column">
-                <h5 class="sample-title">Avatar</h5>
-                <div class="button-sample">
-                    <div class="button-sample">
-                        <igx-avatar igxRipple initials="AZ" [roundShape]="'true'" bgColor="#BDBDBD" color="#e41c77"></igx-avatar>
-                    </div>
-    
-                    <div class="button-sample">
-                        <igx-avatar igxRipple initials="AZ" bgColor="rgb(52, 0, 104)" color="white" size="medium"></igx-avatar>
-                    </div>
-    
-                    <div class="button-sample">
-                        <igx-avatar igxRipple src="assets/images/men/3.jpg" roundShape="true" size="large"></igx-avatar>
-                    </div>
-                </div>
-            </article>
-        </section>
-    
-        <section class="custom-height section">
-            <article class="sample-column">
-                <h5 class="sample-title">Solid Background</h5>
-                <div class="ripple-sample dark" igxRipple>
-                    <p style="margin-bottom:0">Click me to see my ripple!</p>
-                </div>
-            </article>
-        </section>
-    
-    </div>
\ No newline at end of file
+    <section class="section">
```

Comment:
Do we need these wrapping elements?

Reply:
We need most of them in order to display the three examples in one line. I tried to improve the sample.